glean_parser/templates/go_server.jinja2

```diff
@@ -157,7 +159,9 @@ func (g GleanEventsLogger) record(
 	if envelopeErr != nil {
 		panic("Unable to marshal log envelope to json")
 	}
-	fmt.Println(string(envelopeJson))
+	if g.Writer != nil {
+		fmt.Fprintln(g.Writer, string(envelopeJson))
+	}
```

Comment:
In a separate change, I want to remove the `panic`s from this template and instead bubble up errors. As part of that I will also bubble up an error if `g.Writer == nil`